.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+### Description
+
+Please include a summary of the change and what type of change it is (new feature, bug fix, refactoring, documentation).
+Please also include relevant motivation and context.
+List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+### Quick checks:
+
+- [ ] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
+- [ ] There is no other [pull request](https://github.com/ConduitIO/conduit/pulls) for the same update/change.
```

Comment:
```suggestion
- [ ] There is no other [pull request](https://github.com/ConduitIO/conduit-connector-file/pulls) for the same update/change.
```

Reply:
Ah, right, I'll need to change this in other PRs too.